### ISSUE TITLE
Pets With More Personality

### ISSUE THREAD

**Issue by chairbender** [unitystation/unitystation#2583]
Just to capture some random ideas related to pets that someone suggested to me:

1. Pets can have different personalities which affects their behavior, like how much they like to move around the station, whether they like to be around people, how reliably they respond to food / commands, etc...
2. Different breeds of dogs. Each breed might have their own personality?
3. Pets should be able to make sounds, like a dog barking.
4. Pets can do funny / annoying things that some pets do IRL. The sky is the limit here. For example...
   1. if a dog is left in a room and wants to leave but is blocked, it can get frustrated and start barking to be let out. 
   1. Some pets might not be very "potty trained" and might have an "accident". 
   1. A pet might pick up an item and carry it around (no need for a custom inhand, just lerp the item along with the pet).
   1. Retriever behaviors - if you throw something while standing near a pet they'll go grab it and bring it back to you
   1. Pets GTFO when there's a loud sound.

feel free to add. We can split this into smaller tickets if someone wants to take one of these things on.

**Comment by braincrab** [comment]
id be satisfied with the ability to just rename animals that aren't prenamed using something like a labeller.

**Comment by PerfectTangent** [comment]
@corp-0 added a mood system and some general things that mobs can do, so the bare systems for this have been satisfied. Specific mechanics for the AI to do should be broken out into their own issues. We also are planning to replace the ML agents system for a more robust system for NPC navigation, so more NPC stuff should come once that is in place. Closing.
